test/vault_cluster_auth_test.go

```diff
@@ -89,5 +106,30 @@ func testRequestSecret(t *testing.T, terraformOptions *terraform.Options, expect
 	instanceIP := terraform.Output(t, terraformOptions, OUTPUT_AUTH_CLIENT_IP)
 	url := fmt.Sprintf("http://%s:%s", instanceIP, "8080")
 
-	http_helper.HttpGetWithRetry(t, url, 200, expectedResponse, 60, 10*time.Second)
+	http_helper.HttpGetWithRetry(t, url, 200, expectedResponse, 30, 10*time.Second)
+}
+
+func getSyslogs(t *testing.T, terraformOptions *terraform.Options, amiId string, awsRegion string, testName string) {
```

Comment:
FWIW, the file fetching stuff Eugene added is even more powerful: https://github.com/gruntwork-io/terratest/blob/master/modules/aws/ec2-files.go#L119. You can use it to grab syslog, vault logs, and config files from the ASG and write them somewhere on local disk to make debugging easier.

Reply:
Yes, but the client auth instance is not in asg

Reply:
There are methods to fetch from individual EC2 instances in that code too!